src/trigger/src/Subscriber/TriggerSubscriber.php

```
@@ -143,33 +103,90 @@ protected function allEvents(EventDTO $event): void
-                    $args = match ($eventType) {
-                        ConstEventsNames::WRITE->value => [$value],
-                        ConstEventsNames::UPDATE->value => [$value['before'], $value['after']],
-                        ConstEventsNames::DELETE->value => [$value],
-                        default => null,
-                    };
+    /**
+     * Start loop to process events.
+     */
+    protected function loop(): void
+    {
+        $this->chan ??= tap(new Channel($this->channelSize), function (Channel $chan) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$chan'. (undefined)

(UnusedFormalParameter)